### ISSUE TITLE
Some non-intended missing terms from imports

### ISSUE THREAD

**Issue by matentzn** [monarch-initiative/mondo#2034]
I noticed around 150 terms in mondo.owl (48 now that I moved the PRO issue [here](#2206)), which are not imported (and should):

@matentzn 
- [x] ECTO:0000121 
- [x] ECTO:0000330
- [x] ECTO:0001059
- [x] ECTO:0001591
- [x] ExO:0000002
- [x] NBO:0000034
- [x] NBO:0000433
- [x] NBO:0000455
- [x] NBO:0000551
- [x] NBO:0000612
- [ ] NPO:1731
- [ ] NPO:1732
- [ ] NPO:1735
- [ ] NPO:1736
- [x] OBA:1001084

@cmungall 
- [ ] UMLS:C1709838
- [ ] MFOEM:000193
- [x] MFOMD:0000101
- [x] MFOMD:0000105
- [x] MFOMD:0000107
- [x] MFOMD:0000109
- [x] MFOMD:0000119
- [x] MFOMD:0000122

@nicolevasilevsky  
- [ ] NCBITaxon:11046
- [x] FOODON:00003374
- [ ] NCIT:C147564
- [ ] NCIT:C15492
- [ ] NCIT:C15493
- [ ] NCIT:C15496
- [ ] NCIT:C15497
- [ ] NCIT:C16273
- [ ] NCIT:C16326
- [ ] NCIT:C17214
- [ ] NCIT:C19085
- [ ] NCIT:C19683
- [ ] NCIT:C35920
- [ ] NCIT:C36104
- [ ] NCIT:C36869
- [ ] NCIT:C37109
- [ ] NCIT:C37123
- [ ] NCIT:C38328
- [ ] NCIT:C43431
- [ ] NCIT:C45980
- [ ] NCIT:C45981
- [ ] NCIT:C52005
- [ ] NCIT:C53637
- [ ] NCIT:C68748
- [ ] NCIT:C68749

IMO this means that some of the imports have not been properly refreshed:

### Known issue
@matentzn to deal with
- ECTO (#1949)

### Should be refreshed normally
@nicolevasilevsky next time you refresh imports (at release time) ensure that all terms from these namespace are actually imported correctly.
- [x] FOODON 
- [x] NCBITaxon
- [x] MFOEM (not an import (or is it this mf import?) -> _the labels have not been imported_
- [x] MFOMD (same as above?) -> _the labels have not been imported_
- [x] NCIT
- [x] OBA

### Not currently an import: What should we do with these?
@cmungall 
Should we delete terms from this namespace prior to release? Or add them as imports?
- NBO
- ExO
- NPO
- PR
- UMLS

**Comment by cmungall** [comment]
> NBO

I quite like the axioms we have that use NBO, but there are only a handful, I am not sure how we are with things like making a base file for NBO. May be easier to filter for now?

> ExO

Does this come from the ecto import?

> NPO

I'm sure this must come from the ecto import? We needed this way back before we added energy terms to envo

> PR

Switch these to ncbi gene IDs. Maybe make a ticket for this

> UMLS

no idea how these get in here, we should have no logical axioms that reference this in the new mondo.owl, only in imports/equivalencies.owl

ExO
NPO
PR
UMLS

**Comment by matentzn** [comment]
> no idea how these get in here, we should have no logical axioms that reference this in the new mondo.owl, only in imports/equivalencies.owl

```
[Term]
id: MONDO:0042963
name: wandering spleen
...
xref: UMLS:C0272414 {source="MEDGEN:kboom-pr98-c99", source="MONDO:equivalentTo", source="GARD:0000328", source="NCIT:C85224"}
is_a: MONDO:0002332 {source="MESH:D050805", source="linkedlifedata"} ! splenic disease
is_a: UMLS:C1709838
```

Seems like this references the class "Rare Non-Neoplastic Disorder" (which I cant find in the ontology, but maybe someone else can): https://www.ncbi.nlm.nih.gov/medgen/354119

Solution 1: we just delete the `is_a`
Solution 2: We create a Mondo term for "Rare Non-Neoplastic Disorder", xref UMLS:C1709838 to that and replace the is a with the new Mondo term. 

@cmungall your call!

**Comment by nicolevasilevsky** [comment]
action item:
after our recent PRs are merged, check the terms above and then this can be closed.

**Comment by nicolevasilevsky** [comment]
this has been addressed by PR #3966
